python/mdvtools/llm/chat_server_extension.py

```
@@ -62,38 +68,58 @@ def chat_init():
             return {"message": bot.welcome}
-        @project_bp.route("/chat", access_level='editable', methods=["POST"])
-        def chat():
+
+        @socketio.on("chat_request", namespace=f"/project/{project.id}")
+        def chat(data):
             nonlocal bot
-            if not request.json:
-                return {"error": "No JSON data in request"}, 500
-            message = request.json.get("message")
-            id = request.json.get("id")
+            sid = request.sid # type: ignore
```

Flake8 (7.2.0)

[error] at least two spaces before inline comment

(E261)